src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -643,16 +632,7 @@ static List<String> parseDockerCommand(String dockerCommand) {
      */
     @Restricted(NoExternalUse.class)
     static List<String> parseLivenessProbe(String livenessProbeExec) {
-        if (StringUtils.isBlank(livenessProbeExec)) {
-            return null;
-        }
-        // handle quoted arguments
-        Matcher m = SPLIT_IN_SPACES.matcher(livenessProbeExec);
-        List<String> commands = new ArrayList<String>();
-        while (m.find()) {
-            commands.add(substituteEnv(m.group(1).replace("\"", "").replace("?:\\\"", "")));
```

Comment:
Not really sure what `.replace("?:\\\"", "")` is doing. There are no tests for it and it doesn't appear that it would ever match the target char sequence because the last quote character, `"`, would have already been stripped out by the previous `replace`.

Reply:
Initially added via https://github.com/jenkinsci/kubernetes-plugin/pull/158/files but indeed, looks weird.